packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -344,6 +401,19 @@ function printFurtherInstructions(
   console.log(`Visit ${chalk.blue.bold(DOCS_URL)} for the documentation on Expo Modules APIs`);
 }
 
+function printFurtherLocalInstructions(slug: string, name: string) {
+  console.log(`You can now import this module inside your application:`);
+  console.log();
+  console.log(chalk.blue(`import { hello } from '${slug}';`));
+  console.log();
+  console.log(`Visit ${chalk.blue.bold(DOCS_URL)} for the documentation on Expo Modules APIs`);
+  console.log(
+    chalk.yellow(
+      `Remember you need to rebuild your development client or reinstall pods to see the changes.`
```

Comment:
Maybe we can ask the user whether to run `pod install` automatically after all?

Reply:
Yes, but I'd like to split that to a second PR and consider when we want this to happen.